extra-functions/extra-02/script.js

```diff
@@ -0,0 +1,8 @@
+function esParOImpar(number) {
+  if (number % 2 === 0) {
+    return true;
+  }
+  return false;
```

Comment:
same as the other comment, this could be done removing the lines 2, 3, 4, 5 and only doing `return number % 2 === 0`

I prefer the way you did it tho